### PR TITLE
[JENKINS-61224] Removing the "monitor[s] text next to the bell (and color change)

### PR DESCRIPTION
**Pull request by romenrg**
### Description
See [JENKINS-61224](https://issues.jenkins-ci.org/browse/JENKINS-61224).

This PR removes the "monitor[s]" text next to the bell, since as discussed, seems unnecessary and a bit confusing. Other tools (e.g. GitHub) only have the bell and a circle.

EDIT: It also changes the colors of the "monitor" (alert). See comments below.

#### Screenshots
Before on computer:
before-bell-no-text-computer

After on computer:
after-bell-no-text-computer

Before on mobile (already not showing due to responsive behavior):
before-bell-no-text-mobile

After on mobile (no change):
after-bell-no-text-mobile

### Proposed changelog entries

* Remove the "monitor[s]" text next to the bell, as unnecessary, leading to a cleaner UI
* Change the colors of the notifications next to the bell, to make them more noticeable

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] JIRA issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [X] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@fqueiruga 
@batmat 
@jglick
@uhafner 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a JIRA issue should exist and be labeled as `lts-candidate`

### CODE REVIEW

**Comment by batmat** [comment]
@romenrg very cool you were able to work on this!
Could you please provide a screenshot showing the state before/after the change in the description?

Thanks!

**Comment by romenrg** [comment]
@batmat Sure! I've edited the description to add the screenshots.
Very happy to contribute my two cents here :)

**Review by markjacksonfishing — APPROVED** [review]
I tested this and it looks good. Thank you @romenrg

**Review by daniel-beck — CHANGES_REQUESTED** [review]
There are many occurrences of the term left on the UI (e.g. `AdministrativeMonitorsConfiguration` UI), and only addressing this part won't actually resolve the linked issue.

A more exhaustive search should be done, and a new term established, where we cannot go without a label.

**Comment by daniel-beck** [comment]
FWIW I provided the feedback https://github.com/jenkinsci/jenkins/pull/4463#issuecomment-587516396 to the original change in which I say it's OK to make the UI "less red" because the additional width compensates for it. Now we're not red, and not wide.

**Comment by romenrg** [comment]
@daniel-beck To your comments, some thoughts:
- Regarding the occurrences of the term in other places and potential renaming, I think that's a separate topic. I would say this is purely a UI decision, in which we are deciding not to show the text "monitor[s]", but that doesn't imply we are renaming the concept, so I would say that, as of now, the concept remains as "monitor", we have just decided to remove the work from the UI
- Regarding your other comment about the colors, maybe also a separate concern, but if you fee removing the text further de-emphasizes the importance of the notification, I would be happy to invert the colors of the background and the text to make it more noticeable, if you people agree. See the following screenshot of this alternative proposal of the button:
Inverting-colors-notificaiton
(Actually I like this alternative better, red background, white number, as traditionally, but keeping the new rounded shape)

**Comment by daniel-beck** [comment]
@romenrg 

> Regarding your other comment about the colors, maybe also a separate concern?

I mean – not really, both size and color can be used to make the element noticeable. First we replaced color with size, now we take away size.

I like your alternative suggestion a lot. Thanks for that. Let's hope the more design inclined among reviewers agree. 👍 

> Regarding the occurrences of the term in other places and potential renaming, I think that's a separate topic

Unsure. Can probably go either way. Why I think these are connected: While the linked issue calls out that the term is used elsewhere already and calls out that use as _more tolerable_, the problem you're addressing here is identified as the term being misleading ("not intuitive in this context") to users, rather than e.g. an aesthetic choice. But if it's misleading here, it's misleading everywhere.

**Comment by uhafner** [comment]
I like the change 👍 I looks much better now.

**Comment by romenrg** [comment]
@uhafner Thanks for reviewing :). Just to clarify, the change you like is the [original one](https://github.com/jenkinsci/jenkins/pull/4555#issue-385273124) in which I just removed the text and / or [the new colors](https://github.com/jenkinsci/jenkins/pull/4555#issuecomment-596220942) I suggested after discussing with @daniel-beck?

**Comment by uhafner** [comment]
Ah, sorry. Indeed there are two different changes. My intention was to say that I like the badge  without text (I do not use this monitor anyway so for me it just removes some clutter from the layout). 

About the colors: Here I actually would prefer the red background and white text as this is the typical indication of an alert. The current design somehow does not indicate that this should be  an important warning.

**Comment by fqueiruga** [comment]
I say if we are going to go full red background, we should do it right. The current red / white background has a rather bad contrast value (https://contrast-ratio.com/#%23fff-on-%23F94B4E) and it looks even worse agains a non-black header background (like with the ui refresh flag enabled).

I'd say let's explore some options with a better text contrast. One example are bootstrap alerts colors. They offer a great contrast in text against  background (https://contrast-ratio.com/#%23721c24-on-%23f8d7da):

Captura de pantalla 2020-03-09 a las 10 51 55

Captura de pantalla 2020-03-09 a las 10 52 08

These are way easier to read. FTR I'm against choosing the bootstrap colors, but we can do some research and find a color combination that works for Jenkins. WDYT?

**Comment by romenrg** [comment]
@fqueiruga Good point about the constrast.

I do like your proposal of using the [bootstrap "alert-danger"](https://getbootstrap.com/docs/4.4/components/alerts/#link-color) color and background-color for the new UI.

However, for the old UI I think that soft red background is too subtle. I been looking at bootstrap and I think the colors of the ["badge-danger"](https://getbootstrap.com/docs/4.4/components/badge/#pill-badges) are more easy to notice and have a [better contrast](https://contrast-ratio.com/#%23fff-on-%23dc3545) than the original suggestion (not perfect but AA for all font sizes and AAA for "bold above 14pt" which is seems to be the case for this notification. Screenshot:
badge-danger-color

I think it is reasonable to have different colors for the notification in the old UI and in the new UI, since the black vs blue background of the navbar has an impact on how the notification is perceived. What do you people think?

**Review by jglick — APPROVED** [review]

**Comment by daniel-beck** [comment]
I think @romenrg makes good points in his last comment. A slightly different shade of red would have the contrast we want IIUC.

**Comment by Wadeck** [comment]
Two cents on colors, the bootstrap alert-danger colors are nice on white background, but both of our style are completely different from white.

I really like the badge-danger approach Romen proposed for the black one.

**Comment by romenrg** [comment]
I've gone ahead and pushed the color combination that seems to have obtained the broader agreement. The result is now the following:

For the new UI, we are using the bootstrap "alert-danger" set of colors, as proposed by @fqueiruga. See screenshot:
bell-no-text-color-newUI

While for the old UI, we are using the bootstrap "badge-danger" combination of colors, as I proposed to address @daniel-beck 's concern about the "monitor" not receiving enough attention. See screenshot:
bell-no-text-color-oldUI

I'm quite happy with the result. I think the combination of removing the text and using these new sets of colors is quite nice, both in the old and in the new UI :)

**Comment by uhafner** [comment]
- The badge-danger red for the old layout 👍 
- The alert-danger red for the new layout 👎: it is not part of the new color palette and seems not to fit to the blue color in my eyes

But anyway: I hope that both colors are part of theming so they can be replaced with the matching colors of a theme

**Comment by daniel-beck** [comment]
Unsure about the refreshed UI, but the classic one is a lot better now 👍 

> Screenshot 2020-03-11 at 06 51 48
> Screenshot 2020-03-11 at 06 51 58
> Screenshot 2020-03-11 at 06 52 51
> Screenshot 2020-03-11 at 06 53 04

**Review by daniel-beck — APPROVED** [review]

**Comment by daniel-beck** [comment]
I played around with a red background on the refreshed UI, but it has the usual blue/red problem of looking messed up (TIL that is called chromostereopsis). So that probably needs a different approach.

**Comment by batmat** [comment]
@josephbrueggen can you please chime in on the color palette issue raised by Ulli? Thanks

**Review by oleg-nenashev — APPROVED** [review]
It looks good to me, thanks! Putting on hold as per @batmat's review request

**Comment by batmat** [comment]
@uhafner in the meantime, I would love if you can clarify your stance: are you actually requesting a change here? If so can you please do it so using the GitHub feature 
image

If you're actually fine with this change, but would mainly like us to double-check this even after a merge, that would be good to know too. 
Danke!

File: war/src/main/less/abstracts/colors.less
Comment (fqueiruga) [inline war/src/main/less/abstracts/colors.less]:
```suggestion
@monitor-bg-v2: #f8d7da;
```

File: war/src/main/less/abstracts/colors.less
Comment (fqueiruga) [inline war/src/main/less/abstracts/colors.less]:
```suggestion
@monitor-color-v2: #721c24;
```

File: war/src/main/less/ui-refresh-overrides.less
Comment (fqueiruga) [inline war/src/main/less/ui-refresh-overrides.less]:
```suggestion
        color: @monitor-color-v2;
```

File: war/src/main/less/ui-refresh-overrides.less
Comment (fqueiruga) [inline war/src/main/less/ui-refresh-overrides.less]:
```suggestion
        background-color-v2: @monitor-bg;
```

**Review by fqueiruga — APPROVED** [review]
These changes are suggestions to keep variable naming consistent

**Comment by uhafner** [comment]
No, I do not request a change here (we can change some colors later on if required).

**Comment by oleg-nenashev** [comment]
I will merge it in 24 hours if no negative feedback.
Thanks @romenrg !

**Comment by josephbrueggen** [comment]
Thanks @romenrg for the contribution here and also @uhafner for being mindful of the color palette. In the future we may return to address color consistency, but from my perspective this is ready to merge. If we do return to make a minor color adjustment, we will prioritize the following:

- sufficient visual contrast _and_
- ensuring a sufficient sense of urgency since these warnings are often critical in nature

Regardless, let's merge this—thanks everyone!

**Comment by romenrg** [comment]
Thanks everyone for the discussions and feedback. Happy to see this merged now :)

BTW, I've just edited the PR description to add a second "proposed changelog entry" about the color change that we ended up introducing here. Not sure if the changelog entries are processed manually or automatically, so I'm adding this comment to let you know about it.

**Pull request merged**
